visualizationTools:
  ros__parameters:
    overallMapVoxelSize: 0.5
    exploredAreaVoxelSize: 0.3
    exploredVolumeVoxelSize: 0.5
    transInterval: 0.2
    yawInterval: 10.0
    overallMapDisplayInterval: 2
    exploredAreaDisplayInterval: 1
    test_param: true